docs/src/index.md

```diff
@@ -0,0 +1,118 @@
+# GeoMakie.jl
+GeoMakie.jl is a Julia package for plotting geospatial data on a given map projection. It is based on the [Makie.jl plotting ecosystem](https://makie.juliaplots.org/stable/).
+
+The package [ClimateBase.jl](https://juliaclimate.github.io/ClimateBase.jl/dev/) builds upon GeoMakie.jl to create a seamless workflow between analyzing/manipulating climate data, and plotting them.
```

Comment:
It would be beneficial to name other packages that might will benefit from GeoMakie [some more from JuliaGeo]

Reply:
I'm not sure if it is beneficial to speculate in a documentation. What I mention here is very much already possible, and in fact the reason I'm trying to push this PR is exactly so I can tag the new version of ClimateBase.jl that integrates with GeoMakie.jl. That being said, I of course do not oppose at all mentioning more packages here, so please do feel free and suggest something concrete I can write.

Reply:
>workflow between analyzing/manipulating  
as for  manipulating data, GeoJSON and GeoInterface are well integrated with GeoMakie. Definitely, no the same as to `builds upon`. In any case, the intention here be able to connect GeoMakie with other geo-spatial libraries.